src/zc.adoc

```diff
@@ -0,0 +1,2616 @@
+|===
+|RV32
+|RV64
+|Mnemonic
+|Instruction
+
+|yes     
+|yes     
+|c.lbu _rd'_, uimm(_rs1'_)
+|<<#insns-c_lbu>>
+
+|yes     
+|yes     
+|c.lhu _rd'_, uimm(_rs1'_)
+|<<#insns-c_lhu>>
+
+|yes     
+|yes     
+|c.lh _rd'_, uimm(_rs1'_)
+|<<#insns-c_lh>>
+
+|yes     
+|yes     
+|c.sb _rs2'_, uimm(_rs1'_)
+|<<#insns-c_sb>>
+
+|yes     
+|yes     
+|c.sh _rs2'_, uimm(_rs1'_)
+|<<#insns-c_sh>>
+
+|yes     
+|yes     
+|c.zext.b _rsd'_
+|<<#insns-c_zext_b>> 
+
+|yes     
+|yes     
+|c.sext.b _rsd'_
+|<<#insns-c_sext_b>> 
+
+|yes     
+|yes     
+|c.zext.h _rsd'_
+|<<#insns-c_zext_h>> 
+
+|yes     
+|yes     
+|c.sext.h _rsd'_
+|<<#insns-c_sext_h>> 
+
+|
+|yes     
+|c.zext.w _rsd'_
+|<<#insns-c_zext_w>> 
+
+|yes     
+|yes     
+|c.not _rsd'_
+|<<#insns-c_not>> 
+
+|yes     
+|yes     
+|c.mul _rsd'_, _rs2'_
+|<<#insns-c_mul>> 
+
+|===
+
+<<< 
+
+[#Zcmp]
+=== Zcmp
+
+The Zcmp extension is a set of instructions which may be executed as a series of existing 32-bit RISC-V instructions.
+
+This extension reuses some encodings from _c.fsdsp_.  Therefore it is _incompatible_ with <<Zcd>>,
+ which is included when C and D extensions are both present. 
+
+NOTE: Zcmp is primarily targeted at embedded class CPUs due to implementation complexity. Additionally, it is not compatible with architecture class profiles.
+
+The Zcmp extension depends on the <<Zca>> extension.
+
+The PUSH/POP assembly syntax uses several variables, the meaning of which are:
+
+* _reg_list_ is a list containing 1 to 13 registers (ra and 0 to 12 s registers)
+** valid values: {ra}, {ra, s0}, {ra, s0-s1}, {ra, s0-s2}, ..., {ra, s0-s8}, {ra, s0-s9}, {ra, s0-s11}
+** note that {ra, s0-s10} is _not_ valid, giving 12 lists not 13 for better encoding
+* _stack_adj_ is the total size of the stack frame. 
+** valid values vary with register list length and the specific encoding, see the instruction pages for details.
+
+[%header,cols="^1,^1,4,8"]
+|===
+|RV32
+|RV64
+|Mnemonic
+|Instruction
+
+|yes     
+|yes     
+|cm.push _{reg_list}, -stack_adj_
+|<<#insns-cm_push>> 
+
+|yes     
+|yes     
+|cm.pop _{reg_list}, stack_adj_
+|<<#insns-cm_pop>> 
+
+|yes     
+|yes     
+|cm.popret _{reg_list}, stack_adj_
+|<<#insns-cm_popret>> 
+
+|yes     
+|yes     
+|cm.popretz _{reg_list}, stack_adj_
+|<<#insns-cm_popretz>> 
+
+|yes     
+|yes     
+|cm.mva01s _rs1', rs2'_
+|<<#insns-cm_mva01s>> 
+
+|yes     
+|yes     
+|cm.mvsa01 _r1s', r2s'_
+|<<#insns-cm_mvsa01>> 
+
+|===
+
+<<<
+
+[#Zcmt]
+=== Zcmt
+
+Zcmt adds the table jump instructions and also adds the JVT CSR. The JVT CSR requires a 
+state enable if Smstateen is implemented. See <<csrs-jvt>> for details.
+
+This extension reuses some encodings from _c.fsdsp_.  Therefore it is _incompatible_ with <<Zcd>>,
+ which is included when C and D extensions are both present. 
+
+NOTE: Zcmt is primarily targeted at embedded class CPUs due to implementation complexity. Additionally, it is not compatible with architecture class profiles.
```

Comment:
```suggestion
NOTE: Zcmt is primarily targeted at embedded class CPUs due to implementation complexity. Additionally, it is not compatible with RVA profiles.
```

The A in RVA stands for Application, not Architecture.

Reply:
Thanks